esp-hal/src/timer/systimer.rs

```diff
@@ -496,18 +400,6 @@ pub trait Comparator {
         }
     }
 
-    /// Get the actual target value of the comparator.
-    fn actual_target(&self) -> u64 {
```

Comment:
We probably want to make this part of the `Timer` trait instead of an inherent method on alarm, I'm conscious in this PR to only expose systimer specific things publicly